### ISSUE TITLE
Email field should not be optional for human users

### ISSUE THREAD

**Issue by DavideWalder** [zitadel/terraform-provider-zitadel#11]
The email field is declared as optional:
https://github.com/zitadel/terraform-provider-zitadel/blob/97b2d6c6cf46906ae4a15542bdf1b77bfba129f7/zitadel/v2/user.go#L105

But omitting it triggers this error:
```
Error: failed to create human user: rpc error: code = InvalidArgument desc = invalid AddHumanUserRequest.Email: value is required
```

**Comment by stebenz** [comment]
Totally missed that, I will have a look at it.

**Comment by stebenz** [comment]
corrected in v1.0.0-alpha.8
